check_limits.py

```diff
@@ -1,18 +1,12 @@
 
+def is_temperature_out_of_range(temperature):
+    return temperature < 0 or temperature > 45
+def is_soc_out_of_range(soc):
+    return soc < 20 or soc > 80
+def is_charge_rate_out_of_range(charge_rate):
+    return charge_rate > 0.8
 def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
-
-  return True
-
-
+    return not (is_temperature_out_of_range(temperature) or is_soc_out_of_range(soc) or is_charge_rate_out_of_range(charge_rate))
```

Comment:
who is responsible to print status of the battery and parameter validation status message ?